source/includes/mapping-options-preview.rst

```diff
@@ -1,6 +1,12 @@
-The blue dotted box in the MongoDB diagram pane is a visual preview of how each 
-rule and option affect your destination data model. Using this feature, 
-you can preview your destination data model before you run a sync job.
+You can preview how mapping rules affect your data model by viewing the 
+blue dotted box in the MongoDB diagram pane. This allows you to preview 
+the mapping rule before you run a sync job.
```

Comment:
Maybe `before you apply it` as there are quite a few steps in between creating the mapping and running a job?

Reply:
Sure that makes sense, updated.